html/semantics/popovers/WEB_FEATURES.yml

```diff
@@ -4,3 +4,7 @@ features:
 - name: closewatcher
   files:
   - "popover-close-request.html"
+- name: backdrop
```

Comment:
I don't think the popover tests should be included in the backdrop feature. It's a somewhat tricky case, because `::backdrop` doesn't exist on its own but only plays a supporting role for other features, originally fullscreen and `<dialog>`, and IIRC the history/evolution was a bit messy, so it's not straightforward to say which of those features got `::backdrop` first.

But features that came much later and use `::backdrop` are more akin to a new feature that uses promises. I think that popover uses backdrop, but doesn't really change the feature.